src/metabase/models/setting.clj

```diff
@@ -323,7 +325,11 @@
    ;; should be used for most non-sensitive settings, and will log the value returned by its getter, which may be a
    ;; the default getter or a custom one.
    ;; (default: `:no-value`)
-   :audit       (s/maybe (s/enum :never :no-value :raw-value :getter))})
+   :audit       (s/maybe (s/enum :never :no-value :raw-value :getter))
+
+   ;; TODO: make this required and deprecate setting/exported-setting
```

Comment:
Maybe that should happen in #37640?

Reply:
Chris wrote in the description of that PR that he'll make it required in a third PR. 

> The addition of explicit :export? false annotations, and making the key a required one, will be done in a subsequent PR.

Reply:
Ah, right, description has merged in my head and I imagined the change coming in as two PRs in total. :)